docs/operator-manual/applicationset/Generators-Pull-Request.md

```diff
@@ -276,6 +276,45 @@ spec:
 * `tokenRef`: A `Secret` name and key containing the Azure DevOps access token to use for requests. If not specified, will make anonymous requests which have a lower rate limit and can only see public repositories. (Optional)
 * `labels`: Filter the PRs to those containing **all** of the labels listed. (Optional)
 
+## SCM-Manager
+
+Fetch pull requests from a repo hosted on a SCM-Manager Server. The Review plugin must be installed on your SCM-Manager to enable pull requests.
+
+```yaml
+apiVersion: argoproj.io/v1alpha1
+kind: ApplicationSet
+metadata:
+  name: myapps
+spec:
+  generators:
+  - pullRequest:
+      scmManager:
+        # Repository Namespace. Required.
+        namespace: myproject
+        # Repository Name. Required.
+        repo: myrepository
```

Comment:
I think this needs to be `name`. If it is supposed to be `repo`, can you update below description of each parameter to use `repo`?

Reply:
You're definitely right. Fixed this. 👍🏽